docs/content/docs/_index.md

```diff
@@ -1229,6 +1229,9 @@ Marks a variable as safe: HTML will not be escaped anymore.
 Accesses a value from an object when the key is not a Tera identifier.
 Example: `{{ sections | get(key="posts/content") }}`
 
+The `get` filter also has a `default` parameter which can be used to provide a return value when the `key` parameter is missing from the set being filtered.
+Example: `{{ sections | get(key="posts/content", default=null) }}`
```

Comment:
does that work? I don't think Tera itself has the concept of null in the template? It's something that should be fixed in v2 though

Reply:
You are correct. I'd defined that in my own code and just forgot it wasn't built in.